Libplanet.RocksDBStore/MonoRocksDBStore.cs

```diff
@@ -543,6 +544,39 @@ public override TxExecution GetTxExecution(BlockHash blockHash, TxId txid)
             return null;
         }
 
+        /// <inheritdoc cref="BaseStore.PutTxIdBlockHashIndex(Guid, TxId, BlockHash)"/>
+        public override void PutTxIdBlockHashIndex(Guid chainId, TxId txId, BlockHash blockHash)
+        {
+            var cf = GetColumnFamily(_chainDb, chainId);
+            _chainDb.Put(
```

Comment:
if we decide to store these indexes on `chainDb`, we also need to add a new method for forking because we can delete the previous `chainDb` after forking... 

Or, how about using the separated DB for storing?

Reply:
I was thinking that It(`forking`) is handled by other codebases. 
And I realized that the chain id may not be needed. So the `forking` also not needed.
please see the patch 5f43072

Reply:
Yes, if we don't put it with `ColumnFamily`, there is no matter. 😄